docs/ref/contrib/admin/index.txt

```diff
@@ -1332,6 +1332,19 @@ subclass::
     :meth:`ModelAdmin.get_search_results` to provide additional or alternate
     search behavior.
 
+    .. versionchanged:: 4.1
+
+        Searches employing multiple search terms over a single reverse foreign
+        key or many-to-many relationship changed to only select rows from the
+        related model where each term matches some certain row, following the
+        first example query in
+        :ref:`topic-spanning-multi-valued-relationships`. For example, if 
+        ``search_fields`` is set to ``['child__name', 'child__age']``, and a
+        user searches for ``Jamal 17``, parent rows will be returned only if
+        there is a relationship to some 17-year-old child named Jamal, rather
+        than also returning parents who merely have a younger or older child
+        named Jamal in addition to some other 17-year-old.
+
```

Comment:
I think the phrasing about the _first example somewhere else_ is difficult. 

Can we rephrase? 

Maybe something along the lines of... 

```suggestion
    .. versionchanged:: 4.1
    
        Searches for multiple search terms are now applied in a single call to
        `filter()`, rather than in sequential `filter()` calls.

        This means that results will be returned for records matching all terms
        rather than any term. For example, if ``search_fields`` is set to
        ``['child__name', 'child__age']``, and a user searches for ``Jamal
        17``, parent rows will be returned only if there is a relationship to
        some 17-year-old child named Jamal, rather than also returning parents
        who merely have a younger or older child named Jamal in addition to
        some other 17-year-old.

        See the :ref:`topic-spanning-multi-valued-relationships` topic for more
        discussion of this difference. 
```

Reply:
I like it. I just plan to clarify that the change is only for multi-valued relationships; vanilla search terms already were subject to "results will be returned for records matching all terms rather than any term"

I'll change throughout, but feel free to ask for changes!

Reply:
Thanks @jacobtylerwalls! 

> ... I just plan to clarify...

Please do! It was _ad lib_ so you may be able to be more precise for sure. 🙂

Reply:
done!